Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -990,15 +990,23 @@ public override bool CanConvert(Type objectType)
         }
 
         /// <summary>
-        /// Distributed Tracing Options. <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/>
-        /// </summary>
-        internal DistributedTracingOptions DistributedTracingOptions { get; set; }
-
+        /// Distributed Tracing Options. <see cref="Microsoft.Azure.Cosmos.DistributedTracingOptions"/> are only applicable when EnableDistributedTracing flag is true.
```

Comment:
This needs a bit of polishing>

```suggestion
        /// Allows for customization of Distributed Tracing.
```

The link with EnableDistributedTracing should be a remark

```
<remark>Customization of Distributed Tracing is allowed when <see cref="Microsoft.Azure.CosmosClientOptions.EnableDistributedTracing"/> is true</remark>
```